entries/o/optumbank.com.json

```diff
@@ -2,13 +2,11 @@
   "Optum Bank": {
     "domain": "optumbank.com",
     "url": "https://www.optumbank.com",
-    "contact": {
-      "facebook": "myOptum",
-      "twitter": "myOptum"
-    },
-    "regions": [
-      "us"
```

Comment:
The regions array is removed. Was this a mistake?

Reply:
Re-added